src/app/query/bitcoin/stamps/stamps-by-address.query.ts

```diff
@@ -68,7 +77,12 @@ async function fetchStampsByAddress(address: string): Promise<StampsByAddressQue
   const resp = await axios.get<StampsByAddressQueryResponse>(
     `https://stampchain.io/api/v2/balance/${address}`
   );
-  return resp.data;
+  try {
+    return stampsByAdressSchema.parse(resp.data);
+  } catch (e) {
+    if (e instanceof ZodError) void analytics.track('schema_fail', e);
+    throw e;
```

Comment:
@kyranjamie won't just throwing an error here be a bad UX for users?

Reply:
Throwing an error here is the same as the request failing with a 400, so it just doesn't work (better than returning unknown data that breaks app)

Reply:
Ok, thx, I was thinking it would break the UI, nvm.